spec/features/resource_manager_manages_holidays_spec.rb

```diff
@@ -92,64 +119,112 @@ def when_they_view_holidays
   end
 
   def then_they_can_see_the_holidays_for_this_week
-    expect(@page.all_events).to eq [
-      {
-        title: 'First user holiday - First Guider',
-        time: '9:00 - 11:00'
-      },
-      {
-        title: 'Merged Holiday - First Guider, Second Guider',
-        time: '13:00 - 14:00'
-      }
-    ]
+    holidays = @page.calendar.holidays
+
+    expect(holidays.count).to eq 2
+
+    expect(holidays.first[:title]).to eq 'First user holiday'
+    expect(holidays.first[:start]).to eq '2016-10-18T09:00:00.000Z'
+    expect(holidays.first[:end]).to eq '2016-10-18T11:00:00.000Z'
+
+    expect(holidays.second[:title]).to eq 'Merged Holiday'
+    expect(holidays.second[:start]).to eq '2016-10-18T13:00:00.000Z'
+    expect(holidays.second[:end]).to eq '2016-10-18T14:00:00.000Z'
   end
 
   def and_they_can_see_the_holidays_for_next_week
     @page.next_week.click
-    expect(@page.all_events).to eq [
-      {
-        title: 'Christmas',
-        time: '9:00 - 23:59'
-      },
-      {
-        title: 'Second user holiday - Second Guider',
-        time: '9:00 - 14:00'
-      }
-    ]
+    holidays = @page.calendar.holidays
+
+    expect(holidays.count).to eq 2
+
+    expect(holidays.first[:title]).to eq 'Second user holiday'
+    expect(holidays.first[:start]).to eq '2016-10-25T09:00:00.000Z'
+    expect(holidays.first[:end]).to eq '2016-10-25T14:00:00.000Z'
+
+    expect(holidays.second[:title]).to eq 'Christmas'
+    expect(holidays.second[:start]).to eq '2016-10-25'
+    expect(holidays.second[:end]).to eq '2016-10-27'
   end
 
   def and_there_is_a_guider
     @guider = create(:guider)
   end
 
-  def and_they_create_a_holiday
+  def and_they_create_a_multiple_day_holiday
     @page.wait_until_create_holiday_visible
 
     @page.create_holiday.click
     @page = Pages::NewHoliday.new
 
+    @page.create_a_multiple_day_holiday.set(true)
+    @page.title.set 'Holiday Title'
+    start_at = I18n.l(Time.zone.now.to_date, format: :date_range_picker)
+    end_at = I18n.l(Time.zone.now.to_date + 2.days, format: :date_range_picker)
+    @page.all_day_date_range.set "#{start_at} - #{end_at}"
+    page.execute_script '$(".daterangepicker").hide();'
```

Comment:
Can this not be executed in the context of the `@page`? We can push it into the page object as it's being used multiple times around these specs.